### PR TITLE
SAVAS-Exercise 31 was completed

### PR DESCRIPTION
**Pull request by savaspamuk**

### CODE REVIEW

File: JS/Lesson31/savas/my-app/src/components/NewItem/index.jsx [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+    const handleSubmit = (event) => {
+      event.preventDefault();
+      if (item) {
+        handleNewTodo(item);
+        setItem("");
+      }
+    };
+  
+    return (
+      <>
```
Comment (annasvst) [inline JS/Lesson31/savas/my-app/src/components/NewItem/index.jsx:18]: No need for fragment component (<>) here as you only have a <form> component on the most outer layer in the return statement

Comment (annasvst) [inline JS/Lesson31/savas/my-app/src/components/NewItem/index.jsx:18]:
```suggestion
```

File: JS/Lesson31/savas/my-app/src/components/NewItem/index.jsx [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+            value={item}
+            onChange={handleInputChange}
+            type="text"
+          />
+          <button type="submit">
+            Add new task
+          </button>
+          <hr />
+        </form>
+      </>
```
Comment (annasvst) [inline JS/Lesson31/savas/my-app/src/components/NewItem/index.jsx:30]:
```suggestion
```

**Review by annasvst — COMMENTED** [review]
Looks good Savaş! I've added a minor comment about usage of Fragment component, please read more here: https://react.dev/reference/react/Fragment
Everything else looks good. Let me know once suggestion is resolved and I'll approve. :)

File: JS/Lesson31/savas/my-app/src/components/NewItem/index.jsx (hunk 1 above)
Comment (savaspamuk) [reply JS/Lesson31/savas/my-app/src/components/NewItem/index.jsx]: Oh, I missed it :)

**Review by annasvst — APPROVED** [review]
Approved!

**Pull request merged**